proxy/src/main/java/com/wavefront/agent/data/EntityProperties.java

```diff
@@ -181,5 +181,21 @@
      * @param traceSamplingRate sampling rate for tracing spans
      */
     void setTraceSamplingRate(Double traceSamplingRate);
+
+    /**
+     * Get the maximum delay between the end time of a tracing span to reporting to wavefront,
+     * beyond which they are dropped.
+     *
+     * @return delay threshold for dropping spans in minutes.
+     */
+    Integer getDropSpansDelayedMinutes();
+
+    /**
+     * Set the maximum delay between the end time of a tracing span to reporting to wavefront,
```

Comment:
Set the maximum acceptable duration between now and the end of a span to be accepted for reporting to Wavefront, beyond which they are dropped.